nuget.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
```

Comment:
Do we need this `<clear>`? Would this obliterate any feeds in my user-scoped `nuget.config` file. Some feeds currently in my file:
```
<!-- local nupkg files used for testing purposes -->
<add key="local" value="C:\Users\lucas.pimentel\Downloads\nuget-packages" />

<!-- added by VS -->
<add key="Microsoft Visual Studio Offline Packages" value="C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\" />

<-- our feed in AzDO, used ocasionally -->
<add key="Azure DevOps Public Feed" value="https://pkgs.dev.azure.com/datadoghq/dd-trace-dotnet/_packaging/Public_Feed/nuget/v3/index.json" />

<-- added by Service Fabric SDK -->
<add key="Microsoft Azure Service Fabric SDK" value="C:\Program Files\Microsoft SDKs\Service Fabric\packages" />
```

Reply:
Yes and no. This _will_ blat user-scoped nuget.config, but that was partly intentional to make things reproducible. It also avoids any naming clashes, but I can work around that by providing a random name, and seeing what happens when a feed is added twice 🤔

IMO, if new feeds are required (e.g. service fabric) they should be included here, so it's reproducible. The temp local packages one is something I do too, but normally just include it temporarily. 

Nevertheless, I'll try removing the `<clear />` and see how it goes

Reply:
Don't worry too much about it. I can always remove the `<clear>` locally when needed.